executor/batch_point_get.go

```diff
@@ -349,6 +359,9 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	e.values = make([][]byte, 0, len(values))
 	for i, key := range keys {
 		val := values[string(key)]
+		// track memory
+		e.memTracker.Consume(int64(len(key)) + int64(len(val)))
```

Comment:
I think the "memory" we are tracking, should be the "memory" consumed by executor in its life time.

IMO, for `BatchPointGetExec`, `e.handle`, `e.values`, `e.idxVals` are such members "consuming" significant memory. While `key` & `keys` are not, which would be released after `initialize()` return.

So I suggest tracking memory usage for these member variables.

Besides, slices would use memory more than actual length. Memory for capacity of slice would be more accurate.

Reply:
done